source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputDateRenderer.mm

```diff
@@ -39,6 +39,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 
     ACRInputLabelView *inputLabelView = [[ACRInputLabelView alloc] initInputLabelView:rootView acoConfig:acoConfig adptiveInputElement:dateInput inputView:dateField accessibilityItem:dateField.inputView viewGroup:viewGroup dataSource:nil];
     dateField.accessibilityTraits = UIAccessibilityTraitButton | UIAccessibilityTraitStaticText;
+    dateField.accessibilityHint = NSLocalizedString(@"opens the date picker", nil);
```

Comment:
at some point localization is going to come for us D:

Reply:
it will; i filed for the exception for that reasons; but we never got around the issue while we were in the grace period.